### PR TITLE
`man` bufname should indicate man page.

### PR DESCRIPTION
**Pull request by nochiel**
- makes `man` buffers easily identifiable,  
- ensures that a previously opened page remains open if a second `man` buffer is opened,
- makes `man` bufname similar to `doc` buffer bufname.

### CODE REVIEW

**Comment by mawww** [comment]
Sorry, I just realized I had a pending review message on that that I never submitted.

File: rc/core/man.kak [hunk 1]
```diff
@@ -27,25 +27,23 @@ hook global WinSetOption filetype=(?!man).* %{
...
     colout=$(mktemp "${TMPDIR:-/tmp}"/kak-man-XXXXXX)
-    MANWIDTH=${kak_window_width} man "$@" > $manout 2>/dev/null
+    MANWIDTH=${kak_window_width} eval man "$@" > $manout 2>/dev/null
     retval=$?
     col -b -x > ${colout} < ${manout}
     rm ${manout}
     if [ "${retval}" -eq 0 ]; then
         printf %s\\n "
-                edit -scratch '$buffer_name'
+                edit -scratch '*man $**'
```
[the remark this answers is not in the document]
Comment (nochiel) [reply rc/core/man.kak]: Thank you.  Fixed.

[the remark this answers is not in the document]
Comment (mawww) [reply rc/core/man.kak]: is it ? I still see the same diff here on github.

[the remark this answers is not in the document]
Comment (nochiel) [reply rc/core/man.kak]: I apologise. Technical problems on my part. I've pushed again.

**Comment by mawww** [comment]
Could you squash those two commit together ?

**Comment by mawww** [comment]
This feature is now in, even though I am still unsure its really better.

**Pull request closed without merging**